src/style.css

```diff
@@ -2,0 +1,77 @@
+@font-face {
+  font-family: Arial;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Arial;
```

Comment:
The font-family name 'Arial' is being redefined in the @font-face rule. Consider using a unique name for the custom font to avoid conflicts with the standard Arial font.